File: check_db.js
const {Pool} = require('mysql2/promise');
(async () => {
  try {
    const pool = createPool({host:'localhost',port:3306,user:'root',password:'',database:'peoplepay360'});
    const [tables] = await pool.query('SHOW TABLES');
    console.log('Tables:', JSON.stringify(tables, null, 2));
    
    // Check working_schedules schema
    try {
      const [schema] = await pool.query('SHOW CREATE TABLE working_schedules');
      console.log('working_schedules schema:', JSON.stringify(schema, null, 2));
    } catch(e) {
      console.log('working_schedules table does not exist or error:', e.message);
    }
    
    // Check contracts schema
    try {
      const [schema] = await pool.query('SHOW CREATE TABLE contracts');
      console.log('contracts schema:', JSON.stringify(schema, null, 2));
    } catch(e) {
      console.log('contracts table error:', e.message);
    }
    
    // Check employees schema
    try {
      const [schema] = await pool.query('SHOW CREATE TABLE employees');
      console.log('employees schema:', JSON.stringify(schema, null, 2));
    } catch(e) {
      console.log('employees table error:', e.message);
    }
    
    await pool.end();
  } catch(e) {
    console.error('Database error:', e);
  }
})();

File: check_db.py
import json
import pymysql

try:
    conn = pymysql.connect(
        host='localhost',
        user='root',
        password='',
        database='peoplepay360',
        charset='utf8mb4'
    )
    cur = conn.cursor()
    
    # Show tables
    cur.execute("SHOW TABLES")
    tables = cur.fetchall()
    print("Tables:", tables)
    
    # Check working_schedules
    for table in tables:
        table_name = table[0]
        print(f"\n=== SHOW CREATE TABLE {table_name} ===")
        cur.execute(f"SHOW CREATE TABLE {table_name}")
        schema = cur.fetchone()
        print(schema[2])
    
    conn.close()
except Exception as e:
    print(f"Error: {e}")